README.md

```diff
@@ -76,72 +76,66 @@ For examples, see [x/examples](./x/examples/).
+This approach is suited for applications that require C bindings. It is similar to the Generated Mobile Library approach, where you need to first create a Go library to generate bindings for.
 
 Steps:
 
-1. **Create a Go library**: Create a Go package that wraps the SDK functionalities you need. Functions to be exported must be marked with `//export`, as per the [cgo documentation](https://pkg.go.dev/cmd/cgo#hdr-C_references_to_Go).
+1. **Create a Go library**: Create a Go package that wraps the SDK functionalities you need. Functions to be exported must be marked with `//export`, as described in the [cgo documentation](https://pkg.go.dev/cmd/cgo#hdr-C_references_to_Go).
 1. **Generate C library**: Use `go build` with the [appropriate `-buildmode` flag](https://pkg.go.dev/cmd/go#hdr-Build_modes). Anternatively, you can use [SWIG](https://swig.org/Doc4.1/Go.html#Go).
 1. **Integrate into your app**: Add the generated C library to your application, according to your build system.
 
 
 ## Tentative Roadmap
 
-The launch will have two milestones: Alpha and Beta. We are currently in Alpha. Most of the code is not new. It's the same code that is currently being used by the production Outline Client and Server. The SDK repackages code from [outline-ss-server](https://github.com/Jigsaw-Code/outline-ss-server) and [outline-go-tun2socks](https://github.com/Jigsaw-Code/outline-go-tun2socks) in a way that is easier to reuse and extend.
+This launch is currently in Beta. Most of the code is not new. It's the same code that is currently being used by the production Outline Client and Server. The SDK repackages code from [outline-ss-server](https://github.com/Jigsaw-Code/outline-ss-server) and [outline-go-tun2socks](https://github.com/Jigsaw-Code/outline-go-tun2socks) in a way that is easier to reuse and extend.
 
-### Alpha
-
-The goal of the Alpha release is to make it available to potential developers early, so they can provide feedback on the SDK and help shape the interfaces, processes and tools.
+### Beta
 
-Alpha features:
+The goal of the Beta release is to make the SDK available for broad consumption, with no major expected changes to the APIs and all supporting resources in place (website, documentation, examples, and so on).
 
-- Transport-level libraries
-  - [x] Add generic transport client primitives (`StreamDialer`, `PacketListener` and Endpoints)
-  - [x] Add TCP and UDP client implementations
-  - [x] Add Shadowsocks client implementations
-  - [x] Use transport libraries in the Outline Client
-  - [x] Use transport libraries in the Outline Server
+Beta features:
 
 - Network-level libraries
-  - [x] Add IP Device abstraction
-  - [x] Add IP Device implementation based on go-tun2socks (LWIP)
-  - [x] Add UDP handler to fallback to DNS-over-TCP
-  - [x] Add DelegatePacketProxy for runtime PacketProxy replacement
+  - [x] Add IP Device abstraction (v 0.0.2)
```

Comment:
I think in the Go world people use the notation `v0.0.2` more often.

Reply:
Good to know, thanks! I'll fix that and request another pull.